src/controllers/simulationcontroller.js

```diff
@@ -58,9 +58,11 @@ class SimulationController extends Controller
     _runHour()
     {
         const demandGenerator = new DemandController();
+        const levelController = new LevelController();
 
         demandGenerator.doCustomerOrderGeneration();
         FactoryController.updateOrder();
+        levelController.checkGoalReached();
```

Comment:
Perhaps run this daily. It does not seem like the kind of thing that needs continuous checks :).